api4/job_test.go

```diff
@@ -23,7 +23,7 @@ func TestCreateJob(t *testing.T) {
 	}
 
 	received, resp := th.SystemAdminClient.CreateJob(job)
-	CheckNoError(t, resp)
+	require.NoError(t, resp.Error)
```

Comment:
Because we are using a custom error, these have to be `require.Nil`'s

Reply:
Understood, fixed.